src/test/auth.request.spec.js

```diff
@@ -0,0 +1,198 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import Helper from '../utils/helpers';
+
+chai.use(chaiHttp);
+chai.should();
+
+const URL_PREFIX = '/api/v1';
+const payload = { id: 1, email: 'cleave@mail.com', is_admin: true };
+const userToken = Helper.genToken(payload);
+
+const requestDetails = {
+  source: ' lagos',
+  destination: ' newyork',
+  tripType: 'return',
+  travelDate: '2019-08-27',
+  returnDate: '2019-09-27',
+  reason: 'business',
+  accommodation: ' Lorem ipsum',
+};
+
+describe('TRIP REQUEST ROUTE', () => {
+  describe('Edit Request', () => {
+    it('should update an open trip request when user is logged in and required details are provided', async () => {
+      const res = await chai.request(app)
+        .patch(`${URL_PREFIX}/request/1`)
+        .set('token', userToken)
+        .send(requestDetails);
+
+      res.should.have.status(200);
+      res.body.should.be.an('object');
```

Comment:
Please, let's assert that a message is equal to the expected message not just expecting an object